src/Microsoft.Health.Fhir.Core/Messages/Conditional/ConditionalResourceRequest.cs

```diff
@@ -14,13 +14,14 @@ namespace Microsoft.Health.Fhir.Core.Messages
 {
     public abstract class ConditionalResourceRequest<TResponse> : IRequireCapability, IRequest<TResponse>
     {
-        protected ConditionalResourceRequest(string resourceType, IReadOnlyList<Tuple<string, string>> conditionalParameters, BundleResourceContext bundleResourceContext)
+        protected ConditionalResourceRequest(string resourceType, IReadOnlyList<Tuple<string, string>> conditionalParameters, bool maxParallelism, BundleResourceContext bundleResourceContext)
```

Comment:
Isn't `maxParallelism` a hint that is only used for Cosmos? This feels like a leaking abstraction here